t/lib/TestDB.pm

```diff
@@ -12,11 +12,37 @@ sub new {
 
     open my $fh, '<', 'share/test_data.sql';
 
+    my $continued_line = "";
+    my $last_line_was_continued = 0;
+
     while (<$fh>) {
         $ENV{DBIC_TRACE}
           and warn $_;
```

Comment:
You left these two lines untouched. Do you know what they do? Set `DBIC_TRACE=1` in your terminal to see any SQL statements that DBIx::Class runs. We need to do this ourselves here because we're using the low-level database handle. `warn "...\n"` with a trailing newline outputs text on STDERR.

Now `$_` is not 1 SQL statement anymore but 1 input line (possibly empty). You can make it dump every SQL statement when you're actually executing it.